models/cable/inst/template.job

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# redirect output
+exec 3>&1
+exec &> "@OUTDIR@/logfile.txt"
+
+# host specific setup
+@HOST_SETUP@
+
+# create output folder
+mkdir -p "@OUTDIR@"
+
+# see if application needs running
+if [ ! -e "@OUTDIR@/results.csv" ]; then
+  cd "@RUNDIR@"
+  
+  "@BINARY@"
+  STATUS=$?
+  
+  # check the status
+  if [ $STATUS -ne 0 ]; then
+    echo -e "ERROR IN MODEL RUN\nLogfile is located at '@OUTDIR@/logfile.txt'" >&3
+    exit $STATUS
+  fi
+
+  # convert to MsTMIP
+  echo "require (PEcAn.CABLE)
```

Comment:
probably better to do PEcAn.CABLE::model2netcdf.CABLE rather than having one line load the library and the next call the function